airship_tempest_plugin/tests/api/shipyard/test_document_staging.py

```
@@ -0,0 +1,159 @@
+            pass
+
+    @decorators.idempotent_id('d42c691a-07a2-41cc-91a6-9d02be8f2649')
+    def test_get_collection_version_committed(self):
+        """Get Config docs of committed version,
+           Successful with response status 200
+        """
+        collection_id = self._get_collection_id()
+        response = self.shipyard_document_staging_client.\
+            get_configdocs_version(collection_id, "committed")
+        self.assertEqual(response.response['status'], '200')
+
+    @decorators.idempotent_id('2940492a-47b0-4218-859a-b9cb556f480b')
+    def test_get_collection_version_successful_site_action(self):
+        """Get Config docs of successful_site_action version,
+        Successful with response status 200
+        """
+        collection_id = self._get_collection_id()
+        response = self.shipyard_document_staging_client.\
+            get_configdocs_version(collection_id, "successful_site_action")
+        self.assertEqual(response.response['status'], '200')
+
+    @decorators.idempotent_id('f6b86b85-5797-4664-95a9-0b6d242b50b7')
+    def test_get_collection_version_last_site_action(self):
+        """Get Config docs of last_site_action,
+           Successful with response status 200
+        """
+        collection_id = self._get_collection_id()
+        response = self.shipyard_document_staging_client.\
+            get_configdocs_version(collection_id, "last_site_action")
+        self.assertEqual(response.response['status'], '200')
+
+    @decorators.idempotent_id('2db31479-8d1f-4d95-a3a0-31b1bf726f39')
+    def test_get_renderedconfigdocs(self):
+        """Get RenderedConfig docs, Successful with response status 200
+           In case document does not found, return 404 Not Found
+        """
+        try:
+            response = self.shipyard_document_staging_client.\
+                get_renderedconfigdocs()
+            self.assertEqual(response.response['status'], '200')
+        except exceptions.NotFound:
```

why are we silencing these exceptions?

---

As per Shipyard API documentation, for some of the API call, there might "404 NotFound" exception be raised, therefore try/exception block has been used. As of now, there are only test cases related to GET API call, once POST API call test cases will be added, setup and teardown will definitely be introduced.